packages/evm/jsonrpc/evmchain.go

```diff
@@ -144,7 +144,7 @@ func (e *EVMChain) iscStateFromEVMBlockNumber(blockNumber *big.Int) (state.State
 		return nil, fmt.Errorf("block number is too large: %s", blockNumber)
 	}
 	// the first EVM block (number 0) is "minted" at ISC block index 1 (init chain)
```

Comment:
This comment is now outdated?

```suggestion
	// the first EVM block (number 0) is "minted" at ISC block index 0
```